xrpl4j-core/src/main/resources/definitions.json

```
@@ -2974,8 +3059,13 @@
     "DIDDelete": 50,
     "OracleSet": 51,
     "OracleDelete": 52,
+    "LedgerStateFix": 53,
+    "MPTokenIssuanceCreate": 54,
+    "MPTokenIssuanceDestroy": 55,
+    "MPTokenAuthorize": 56,
```

See MPT PR [here](https://github.com/XRPLF/rippled/pull/5143/files#diff-6df9171738d4845d658c8714ae8731ed14f7164316871cd8f0e92412d9ff1172R404): 

```suggestion
    "MPTokenIssuanceSet": 56,
```